handler/genshorturl.go

```diff
@@ -0,0 +1,140 @@
+package handler
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"net/http"
+	"open-platform/db"
+	"open-platform/utils"
+
+	"github.com/gin-gonic/gin"
+)
+
+type resultForCreation struct {
+	url  string
+	code string
+	err error
```

Comment:
one more `backspace`, pay attention to the code-style.

You can use the code format tool in the IDE